Dockerfiles/Dockerfile-tools

```diff
@@ -5,12 +5,17 @@ ARG VERSION
 # --------------------------------------------------------------------------------------------------
 # See ./builder for this image
 FROM cytopia/ansible-builder as builder
+ARG VERSION
```

Comment:
You might also need to adjust the `Makefile` to parse the version: https://github.com/cytopia/docker-ansible/blob/master/Makefile#L151